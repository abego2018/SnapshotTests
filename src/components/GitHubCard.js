import Card from 'react-bootstrap/Card'
import Button from 'react-bootstrap/Button'

function GitHubCard(){
    return(
        <Card style={{ width: '18rem' }}>
  <Card.Img variant="top" src="http://placekitten.com/100/100" />
  <Card.Body>
    <Card.Title>Abraham</Card.Title>
    <Card.Text>
      I like to program
    </Card.Text>
    <Button variant="primary">Go somewhere</Button>
  </Card.Body>
</Card>
    )
}

export default GitHubCard