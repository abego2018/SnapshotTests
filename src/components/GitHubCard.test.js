import GitHubCard from "./GitHubCard";
import renderer from 'react-test-renderer'

test ('renders a snapshot', () => {
    const card = renderer.create(<GitHubCard/>).toJSON()
    expect(card).toMatchSnapshot()
  })