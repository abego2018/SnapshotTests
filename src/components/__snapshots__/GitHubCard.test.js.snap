// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="card"
  style={
    Object {
      "width": "18rem",
    }
  }
>
  <img
    className="card-img-top"
    src="http://placekitten.com/100/100"
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      Abraham
    </div>
    <p
      className="card-text"
    >
      I like to program
    </p>
    <button
      className="btn btn-primary"
      disabled={false}
      type="button"
    >
      Go somewhere
    </button>
  </div>
</div>
`;
